app/models/user.rb

```diff
@@ -55,4 +55,21 @@ def create_token
       end while self.class.exists?(registration_token: registration_token)
     end
   end
+
+  def newtask
+   Project.tasks.where("(status = 1 and created_at BETWEEN ? AND ?)" , (DateTime.now - 24.hours), DateTime.now ) 
+  end
+
+  def deletedtask
+    Project.tasks.where("(status = 5 and deleted_at BETWEEN ? AND ?)" , (DateTime.now - 24.hours), DateTime.now )  
+  end
+
+  def confirm_completed
+    User.project.tasks.where("(status = 2  and creator_id != finished_by and updated_at BETWEEN ? AND ? )", (DateTime.now - 24.hours), DateTime.now)
+  end
+
+  def confirm_delete
+     User.project.tasks.where("(status = 4 and creator_id != finished_by and updated_at BETWEEN ? AND ? )", (DateTime.now - 24.hours), DateTime.now)
+  end
+
```

Comment:
Hi @DMips I am running from console : User.find(4).newtask
Is says undefined project. Why is to so ?